### PR TITLE
squashfs-tools: new package at 4.4

### PR DESCRIPTION
**Pull request by eudaldgr**
## Description of package

tools to create and extract Squashfs filesystems

## Installed manifest of package

```
/var/db/kiss/installed/squashfs-tools/version
/var/db/kiss/installed/squashfs-tools/sources
/var/db/kiss/installed/squashfs-tools/manifest
/var/db/kiss/installed/squashfs-tools/depends
/var/db/kiss/installed/squashfs-tools/checksums
/var/db/kiss/installed/squashfs-tools/build
/var/db/kiss/installed/squashfs-tools/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/bin/unsquashfs
/usr/bin/mksquashfs
/usr/bin/
/usr/
```

## New package

- [x] Latest upstream version.
- [x] I agree to maintain this package.
- [x] I agree to notify @dylanaraps if I can no longer maintain this package.

### CODE REVIEW

File: community/squashfs-tools/build [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/sh -e
+
+cd squashfs-tools
+
+# fix build failure against gcc-10
+sed -i "s/struct cache \*bwriter_buffer, \*fwriter_buffer;/extern struct cache \*bwriter_buffer, \*fwriter_buffer;/g"  mksquashfs.h
```
Comment (dylanaraps) [inline community/squashfs-tools/build:6]: No way to make this line shorter?

Comment (eudaldgr) [reply community/squashfs-tools/build]: done

**Comment by dylanaraps** [comment]
Nice stuff

**Comment by eudaldgr** [comment]
Thanks!

**Pull request merged**